File: app/src/main/java/com/soya/launcher/ui/fragment/GuideDateFragment.java
package com.soya.launcher.ui.fragment;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;

public class GuideDateFragment extends AbsDateFragment{
    public static GuideDateFragment newInstance() {

        Bundle args = new Bundle();

        GuideDateFragment fragment = new GuideDateFragment();
        fragment.setArguments(args);
        return fragment;
    }

    @Override
    protected void initBind(View view, LayoutInflater inflater) {
        super.initBind(view, inflater);
        showNext(true);
        showWifi(false);
    }
}

File: app/src/main/java/com/soya/launcher/ui/fragment/GuideDateFragment.kt
package com.soya.launcher.ui.fragment

import android.os.Bundle
import com.shudong.lib_base.base.BaseViewModel
import com.soya.launcher.databinding.FragmentSetDateBinding

class GuideDateFragment : AbsDateFragment<FragmentSetDateBinding,BaseViewModel>() {


    override fun initView() {
        super.initView()
        showNext(true)
    }

    companion object {
        fun newInstance(): GuideDateFragment {
            val args = Bundle()

            val fragment = GuideDateFragment()
            fragment.arguments = args
            return fragment
        }
    }
}
